src/test/cypress/cypress/integration/Acknowledgment.spec.js

```diff
@@ -18,21 +18,26 @@ describe('Acknowledgment  tests', function () {
         // Send a card with ack config set to Never 
         // ack is not possible 
         cy.sendCard('cypress/ack/message4.json');
+
+        // Send a card with ack config set to OnlyWhenResponseDisabledForUser and user can respond and lttd is set
+        // ack is possible when lttd is expired
+        cy.sendCard('cypress/ack/message5.json');
     });
```

Comment:
Add a new test : Send a card with lttd = current time + 2h 
and check it is not possible to ack